articles/integrations/using-templates-sparkpost-wordpress.md

```diff
@@ -30,6 +30,8 @@ To use these variables in your templates, you'll have to use our [Templates API
 }
 ```
 
-Note that content is enclosed in triple curly braces - this allows HTML content to be substituted. In the WordPress admin, on the settings screen for the SparkPost plugin, remember to fill in the Template field (in this example wordpress-testing):
+Note that `content` is enclosed in triple curly braces - this allows HTML content to be substituted without escaping. In the WordPress admin, on the settings screen for the SparkPost plugin, remember to fill in the Template field (in this example wordpress-testing):
```

Comment:
I think the second sentence should be:
```
Remember to fill in the Template field (in this example wordpress-testing) on the settings screen for the SparkPost plugin.
```

If that is too ambiguous and we need to include the "WordPress admin" part:
```
Remember to fill in the Template field (in this example wordpress-testing) on the settings screen for the SparkPost plugin, which can be found in the WordPress admin area.
```